Add templates (#633)
TYPO3-Headless/headless: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a report to help us improve

Reviewer: improve what?

I would rewrite this entire sentence anyway, it has no relation to reporting bugs.

title: ''
labels: bug
assignees: ''

---

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**TYPO3 version and TYPO3 Headless version**

>  Please provide exact version of TYPO3 and TYPO3 Headless

Reviewer: ```suggestion
 Please provide the exact version of TYPO3 and TYPO3 Headless
```

**Additional context**

> Add any other context about the problem here.

Reviewer: ```suggestion
Add any other information related to the problem here.
```
